examples/post_training_quantization/torch_fx/resnet18/main.py

```diff
@@ -14,6 +14,7 @@
 from time import time
 from typing import Tuple
 
+import openvino.torch  # noqa
```

Comment:
Please add description why you need `import openvino.torch`.

Reply:
It is for torch.compile with openvino backend to work. I will add the description.

Reply:
Done

Reply:
It looks like UX issue, because it should be work without this import. Please, open an issue to `openvino` for `torch.compile(..., backend="openvino")`.

Reply:
Alright, I will open the issue